test-app/overlays/dev/kustomization.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: dev
+namePrefix: test-
+commonLabels:
+  instance: test
+
+
+patchesStrategicMerge:
+- storefront-deploy.yaml
+- platform-deploy.yaml
+resources:
+- ../../../common-base
+- cms-content-cm.yaml
+- cms-content-nfs-pvc.yaml
+images:
+- name: virtocommerce/platform
+  newName: docker.pkg.github.com/virtocommerce/vc-platform/platform
+  newTag: dev-3.15.0-alpha.11535-cb44843c
+- name: virtocommerce/storefront
+  newName: docker.pkg.github.com/virtocommerce/vc-demo-storefront/demo-storefront
```

Comment:
Use for test app latest storefront image from vc-storefront repository `docker.pkg.github.com/virtocommerce/vc-storefront/storefront:dev-linux-latest`